bugsnag-android-core/src/main/java/com/bugsnag/android/ObjectJsonStreamer.kt

```diff
@@ -12,7 +13,7 @@ internal class ObjectJsonStreamer {
         internal const val OBJECT_PLACEHOLDER = "[OBJECT]"
     }
 
-    var redactedKeys = setOf("password")
+    var redactedKeys = setOf(Pattern.compile(".*password.*"))
```

Comment:
It's probably worth making this case insensitive, so that we also cover "Password", "passWord", etc.

```suggestion
    var redactedKeys = setOf(Pattern.compile(".*password.*", Pattern.CASE_INSENSITIVE))
```